src/isyntax/isyntax.c

```diff
@@ -1919,10 +1925,9 @@ u32* isyntax_load_tile(isyntax_t* isyntax, isyntax_image_t* wsi, i32 scale, i32
 	i32 valid_offset = (first_valid_pixel * idwt_stride) + first_valid_pixel;
 	u32* bgra = convert_ycocg_to_bgra_block(Y + valid_offset, Co + valid_offset, Cg + valid_offset,
 											tile_width, tile_height, idwt_stride);
-	isyntax->total_rgb_transform_time += get_seconds_elapsed(start, get_clock());
```

Comment:
this and the timing next few lines need to be reverted, unrelated debug changes.

Reply:
for now, yes. I don't see a reason to remove it really, and it can be useful.